src/ingestor.ts

```diff
@@ -62,10 +62,14 @@ export default async function ingestor(req) {
     let aliased = false;
     if (!['settings', 'alias', 'profile'].includes(type)) {
       if (!message.space) return Promise.reject('unknown space');
-      const space = await getSpace(message.space);
-      if (!space) return Promise.reject('unknown space');
-      network = space.network;
-      if (space.voting?.aliased) aliased = true;
+
+      // Disable space checking for follow/unfollow, checking delegated to the writer
+      if (!['follow', 'unfollow'].includes(type)) {
```

Comment:
I think we can include `follow` and `unfollow` in line `63` no?

Reply:
will lose the `message.space` params existence checking